designs/tripleo/osc-tripleo.md

```diff
@@ -89,14 +89,13 @@ outputs:
         docker_config:
         step_1:
           osc:
-            start_order: 0 # TODO: follow up on this value
             # The name of the OSC image in the Kolla container repo**.
             image: &osc_image {get_param: OSCDockerImage}
             # Docker run parameters for OSC***.
-            privileged: false # Cont
+            privileged: false
             net: host # Uses the host network
             detach: false
-            user: osc # TODO: root?
+            user: root
             restart: always
```

Comment:
what is the meaning of restart from Heat Template context?

Reply:
This is the restart policy used by Docker https://docs.docker.com/engine/reference/run/#restart-policies---restart. For instance "This will run the redis container with a restart policy of always so that if the container exits, Docker will restart it"